app/views/like/_like.html.erb

```diff
@@ -1,8 +1,8 @@
 <div class="hidden-print btn-toolbar pull-right" style="margin-left:10px;margin-top:0;">
   <ul class="btn-group">
-        <% if current_user && (current_user.uid == node.uid || current_user.role == "moderator" || current_user.role == "admin") %>
+        <% if current_user && (current_user.uid == node.uid || current_user.role == "moderator" || current_user.role == "admin") || node.has_tag("with:#{current_user.username}") %>
```

Comment:
I believe this has to go inside the `(  )` so that we're sure there is a `current_user` defined before we use one of its properties. That should resolve the errors! Thank you, this is exciting!

Reply:
@jywarren this resolved the errors! That simple bracket missed my eye! Thank you!